ansible_collections/arista/avd/docs/release-notes/3.x.x.md

```diff
@@ -1,9 +1,28 @@
 # Release Notes For Ansible AVD 3.x.x
 
-Documentation for AVD version `3.5.0` [available here](https://www.avd.sh/en/stable/)
+Documentation for AVD version `3.5.0` [available here](https://www.avd.sh/en/v.3.5.0/)
```

Comment:
@ClausHolbechArista - I wasn't sure if we should add link documentation to specific bug releases?